lib/roots/cookbook.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Roots.Cookbook do
+  use Ecto.Schema
```

Comment:
I am glad migrations are working! I am going to practice this on my own so it can be as natural as a rails g migration.